pack/pom.xml

```diff
@@ -51,6 +52,7 @@
             <id>skip-tests</id>
             <properties>
                 <sakai.skip.tests>true</sakai.skip.tests>
+                <sakai.skip.phase>none</sakai.skip.phase>
```

Comment:
And this?

Reply:
This comes from the original vue PR, see https://github.com/sakaiproject/sakai/pull/10484#discussion_r863940978